10-inference.Rmd

```diff
@@ -0,0 +1,806 @@
+--->
+
+### Sampling distributions for means 
+
+In the previous section, our variable of interest&mdash;Timbit flavour&mdash;was
+*categorical*, and the population parameter of interest was the proportion of chocolate
+Timbits. What if we wanted to infer something about a population of *quantitative* variables instead?
+As mentioned in the introduction to this chapter, there are many choices of population parameter
+for each type of observed variable. In this section, we will study the case where we are interested
+in the population mean of a quantitative variable.
+
+In particular, we will look at an example using data from Airbnb, an online
+marketplace for arranging or offering places to stay. The dataset contains
+Airbnb listings for Vancouver, Canada, in September 2020 
+from [Inside Airbnb](http://insideairbnb.com/). 
+Let's imagine (for learning purposes) that our dataset represents the population of all Airbnb rental listings in Vancouver,
+and we are interested in the population mean price per night.
+Our data contains an ID number, neighbourhood,
+type of room, the number of people that the rental accommodates, number of
+bathrooms, bedrooms, beds, and the price per night.
+
+```{r 11-example-means1, echo = FALSE, message = FALSE, warning = FALSE}
+airbnb <- read_csv("data/listings.csv") %>% 
+  select(id, neighbourhood = neighbourhood_cleansed, room_type, accommodates, bathrooms = bathrooms_text, bedrooms, beds,  price) %>% 
+  mutate(price = as.numeric(str_remove(price, "[$]"))) %>% 
+  na.omit()
+airbnb <- airbnb %>% 
+  mutate(id = 1:nrow(airbnb))
+head(airbnb)
+```
+
+We can visualize the population distribution of the price per night with a histogram. 
+```{r 11-example-means2, echo = TRUE, message = FALSE, warning = FALSE, fig.cap = "Population distribution of price per night ($) for all Airbnb listings in Vancouver, Canada"}
+population_distribution <-  ggplot(airbnb, aes(x = price)) +
+    geom_histogram(fill="#0072B2", color="#e9ecef") +
+    xlab("Price per night ($)") 
+population_distribution
+population_parameters <- airbnb %>% 
+    summarize(pop_mean = mean(price),
+             pop_sd = sd(price))
+population_parameters
+```
+
+We see that the distribution has one peak and is skewed&mdash;most of the listings
+are less than \$250 per night, but a small proportion of listings cost more
+than that, creating a long tail on the histogram's right side. 
+The population mean is \$`r round(population_parameters$pop_mean,2)` and the
+population standard deviation is \$`r round(population_parameters$pop_sd, 2)`.
+
+Suppose we take a sample of 20 observations from our population. Below we 
```

Comment:
Why such a small sample size? It might be better/simpler to pick a larger sample size where CLT kicks in, even if we don't talk about it here.

Reply:
changed sample size to 40